src/app/components/PageTitleConfirmation.spec.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { renderToDom } from '../../test/reactutils';
+// import { act } from 'react-dom/test-utils';
+import { initialState } from '../reducer';
+import createTestStore from '../../test/createTestStore';
+import PageTitleConfirmation from './PageTitleConfirmation';
+import TestContainer from '../../test/TestContainer';
+import { setHead } from '../head/actions';
+
+describe('PageTitleConfirmation', () => {
+  it('skips blank, skips first non-blank, announces subsequent', async() => {
+    const state = {
+      ...initialState,
+    };
+
+    state.head.title = 'initial';
+    const store = createTestStore(state);
+
+    store.dispatch(setHead({...initialState.head, title: ''}));
+    const component = renderToDom(
+      <TestContainer store={store}>
+        <PageTitleConfirmation className='any' />
+      </TestContainer>
+    );
+
+    expect(component.node.textContent).toBe('');
+    store.dispatch(setHead({...initialState.head, title: 'initial'}));
```

Comment:
i believe its gonna want the dispatches to be wrapped in `act`

Reply:
No complaints from the test.
```
% yarn jest PageTitleConfirmation.spec
yarn run v1.22.22
$ /Users/rej2/Documents/GitHub/rex-web/node_modules/.bin/jest PageTitleConfirmation.spec
 PASS  src/app/components/PageTitleConfirmation.spec.tsx (6.811 s)
  PageTitleConfirmation
    ✓ skips blank, skips first non-blank, announces subsequent (28 ms)

Test Suites: 1 passed, 1 total
Tests:       1 passed, 1 total
Snapshots:   0 total
Time:        7.778 s
Ran all test suites matching /PageTitleConfirmation.spec/i.
✨  Done in 21.04s.
```